apis/proto/v1beta1/fn/run_function_grpc.pb.go

```diff
@@ -0,0 +1,126 @@
+//
+//Copyright 2022 The Crossplane Authors.
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+// versions:
+// - protoc-gen-go-grpc v1.3.0
+// - protoc             (unknown)
+// source: proto/v1beta1/fn/run_function.proto
+
+package v1beta1
```

Comment:
I wonder whether this belongs here, or in a `crossplane/function-runtime-go` repo?

I would expect the generated Python code to live in `crossplane/function-runtime-python` for example. Go is a bit of a special case because the generated code will be used by c/c and c/function-runtime-oci as well as any Function written in Go.